src/parser.js

```diff
@@ -515,15 +515,15 @@ define([
             return undefined;
         }
         var str = attrString.toLowerCase().replace(/\s/g, '');
-        if (str === 'true()') {
-            return true;
-        } else if (str === 'false()') {
+        if (str === 'false()') {
             return false;
+        } else if (str) {
+            return true;
```

Comment:
why was this change required?

Reply:
the attribute value will no longer necessarily be 'true()' or 'false()'. there are now many other truthy values